tests/integration/revision_test.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 func TestRevisionMonotonicWithLeaderPartitions(t *testing.T) {
-	testRevisionMonotonicWithFailures(t, 11*time.Second, func(clus *integration.Cluster) {
+	testRevisionMonotonicWithFailures(t, 12*time.Second, func(clus *integration.Cluster) {
```

Comment:
Why change 11 to 12? :)

Reply:
The `testRevisionMonotonicWithFailures` function is always called with `11*time.Second`. The lint blames this and it suggests us to use fixed number in the function instead of argument.

```
revision_test.go:77:54: `testRevisionMonotonicWithFailures` - `testDuration` always receives `11 * time.Second (11000000000)` (unparam)
func testRevisionMonotonicWithFailures(t *testing.T, testDuration time.Duration, injectFailures func(clus *integration.Cluster)) {
                                                     ^
```

Most of changes are related to `X` always receives `Y`.

Reply:
thx & interesting